contracts/core/ModuleCore.sol

```diff
@@ -58,7 +56,13 @@ contract ModuleCore is OwnableUpgradeable, UUPSUpgradeable, PsmCore, Initialize,
         return PairLibrary.initalize(pa, ra).toId();
     }
 
-    function initializeModuleCore(address pa, address ra, uint256 lvFee, uint256 initialDsPrice) external override onlyConfig {
+    function initialize(
+        address pa,
+        address ra,
+        uint256 lvFee,
+        uint256 initialDsPrice,
+        uint256 _psmBaseRedemptionFeePercentage
+    ) external onlyConfig {
```

Comment:
please also change the function signature in the interface, don't leave out the override keyword here, it's important that the two match up.(in general also don't rename the function without discussing with others, adding/removing variables is fine)